debug_module.tex

```diff
@@ -80,18 +80,15 @@ \section{Reset Control} \label{reset}
 indicating it's not possible to perform any abstract commands during this time.
 Once a hart's reset is complete, {\tt havereset} becomes set.  When a hart comes out
 of reset and \FdmDmcontrolHaltreq or \Fresethaltreq are set, the hart will
-immediately enter Debug Mode (halted state). Otherwise it will execute normally
-(running state).
+immediately enter Debug Mode (halted state).
+If the hart was initially running it will execute normally (running state).
```

Comment:
Do we need an "otherwise" or "If haltreq and resethaltreq are both clear, then..." before the second part of this?

Reply:
That's probably better.